src/clientiface.cpp

```diff
@@ -384,6 +385,8 @@ void RemoteClient::GetNextBlocks (
 		if (d > full_d_max) {
 			new_nearest_unsent_d = 0;
 			m_nothing_to_send_pause_timer = 2.0f;
+			infostream << "Server: map update for player " << m_name << " (peer " << peer_id << "), time: " << m_map_send_completion_timer << "s" << std::endl;
```

Comment:
apparently we use `o<<"RemoteClient "<<peer_id<<": "` in some other case so that might be better, pick one

also I'd reword it to "full map send completed after Xs, restarting"